pkg/network/ebpf/c/runtime/tracer.c

```diff
@@ -90,6 +91,8 @@ int kretprobe__tcp_sendmsg(struct pt_regs* ctx) {
         return 0;
     }
 
+    update_cgroup_name();
```

Comment:
This seems like something expensive to do on every send/recv. Is there a way to track when cgroups are created/destroyed? Perhaps CWS has some prior work here.